.travis.yml

```diff
@@ -1,43 +1,58 @@
+sudo: required
+
 language: python
 
 services:
-  - postgresql
+  - docker
 
 env:
   global:
-    - DATABASE_URL='postgres://postgres:@localhost:5432/test_db'
-    - SECRET_KEY='tT\xd7\xb06\xf7\x9b\xff\x0fZL\xca\xca\x11\xefM\xacr\xfb\xdf\xca\x9b'
-    - DJANGO_SETTINGS_MODULE=core.settings.test_travis
+    - ON_TRAVIS=True
```

Comment:
Travis Flag. Useful for unittests to avoid some tests being executed on Travis.
@ismailsunni for all your local tests, you can use this environment variable if needed.

Reply:
See also https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables